tests/wpt/runner/expectation.json

```diff
@@ -17828,5 +17828,388 @@
+        "WebKitCSSMatrix stringifier: Infinity (3d)",
+        "WebKitCSSMatrix stringifier: -Infinity (2d)",
+        "WebKitCSSMatrix stringifier: -Infinity (3d)",
+        "WebKitCSSMatrix stringifier: 1/3 (2d)",
+        "WebKitCSSMatrix stringifier: 1/3 (3d)",
+        "WebKitCSSMatrix stringifier: 1/300000 (2d)",
+        "WebKitCSSMatrix stringifier: 1/300000 (3d)",
+        "WebKitCSSMatrix stringifier: 1/300000000 (2d)",
+        "WebKitCSSMatrix stringifier: 1/300000000 (3d)",
+        "WebKitCSSMatrix stringifier: 100000 + (1/3) (2d)",
+        "WebKitCSSMatrix stringifier: 100000 + (1/3) (3d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 1 (2d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 1 (3d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 2 (2d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 2 (3d)",
+        "WebKitCSSMatrix stringifier: Number.MAX_VALUE (2d)",
+        "WebKitCSSMatrix stringifier: Number.MAX_VALUE (3d)",
+        "WebKitCSSMatrix stringifier: Number.MIN_VALUE (2d)",
+        "WebKitCSSMatrix stringifier: Number.MIN_VALUE (3d)",
+        "WebKitCSSMatrix stringifier: throwing getters (2d)",
+        "WebKitCSSMatrix stringifier: throwing getters (3d)"
+      ],
+      "DOMMatrix2DInit-validate-fixup.html": false,
+      "DOMMatrixInit-validate-fixup.html": true,
+      "DOMPoint-001.html": true,
+      "DOMPoint-002.html": true,
+      "DOMQuad-001.html": true,
+      "DOMQuad-002.html": true,
+      "DOMQuad-nan.html": true,
+      "DOMRect-001.html": true,
+      "DOMRect-002.html": true,
+      "DOMRect-nan.html": true,
+      "DOMRectList.html": false,
+      "WebKitCSSMatrix.html": false,
+      "WebKitCSSMatrix.worker.html": true,
+      "historical.html": [
+        "DOMMatrixReadOnly scale number of required arguments",
+        "DOMMatrix scaleSelf number of required arguments",
+        "DOMMatrixReadOnly translate number of required arguments",
+        "DOMMatrixReadOnly scale3d number of required arguments",
+        "DOMMatrixReadOnly rotateFromVector number of required arguments",
+        "DOMMatrixReadOnly rotateAxisAngle number of required arguments",
+        "DOMMatrixReadOnly skewX number of required arguments",
+        "DOMMatrixReadOnly skewY number of required arguments",
+        "DOMMatrix translateSelf number of required arguments",
+        "DOMMatrix scale3dSelf number of required arguments",
+        "DOMMatrix rotateFromVectorSelf number of required arguments",
+        "DOMMatrix rotateAxisAngleSelf number of required arguments",
+        "DOMMatrix skewXSelf number of required arguments",
+        "DOMMatrix skewYSelf number of required arguments",
+        "DOMPointReadOnly matrixTransform number of required arguments",
+        "DOMMatrixReadOnly multiply number of required arguments",
+        "DOMMatrix multiplySelf number of required arguments",
+        "DOMMatrix preMultiplySelf number of required arguments"
```

Comment:
The `length` property of the method needs to be set with the `#[required]` macro.